scripts/apps/templates/controllers/CreateTemplateController.ts

```diff
@@ -10,7 +10,7 @@ import {IArticle, ICustomFieldType, IVocabulary} from 'superdesk-api';
  */
 function applyMiddleware(_item: IArticle, content, vocabularies): Promise<IArticle> {
     // Custom field types with `onTemplateCreate` defined. From all extensions.
-    const fieldTypes: {[id: string]: ICustomFieldType<any, any>} = {};
+    const fieldTypes: { [id: string]: ICustomFieldType<any, any> } = {};
```

Comment:
seems like there are no changes left in this file, can you discard whitespace changes?

Reply:
I meant the entire file. There are still whitespace changes